app/image/image.go

```diff
@@ -19,6 +19,13 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+var (
+	ErrImageNotFound     = errors.New("Image not found")
```

Comment:
Maybe we could define these as structs so we can keep the context? (what image and what version were not found?)

Check this example [here](https://github.com/tsuru/tsuru/blob/master/provision/provision.go#L33).